sosw/components/sns.py

```diff
@@ -122,6 +123,16 @@ def set_separator(self, separator):
         setattr(self, 'separator', separator)
 
 
+    def set_message_attributes(self, message_attributes):
+        """ Validate attribute value and set new value """
+
+        assert isinstance(message_attributes, dict), f"Invalid format of MessageAttributes: {message_attributes}. " \
+            f"MessageAttributes must be a dict"
+        assert len(message_attributes) <= 5, f"Filter policy can have a maximum of 5 attribute names."
```

Comment:
Filter policy - yes. But the message, I believe can have much more custom attributes. And they can be used by other services. Make an integration test with `boto3` anyway.

Reply:
what do you exactly mean by other services?